index.js

```diff
@@ -1 +1,15 @@
 // Your code here
+const findTime = () => { 
+    const date = new Date;
+    const hours = date.getHours()
+    const minutes = date.getMinutes()
+    const seconds = date.getSeconds()
+
+    const formattedHours = (hours < 10 ? '0' : '') + hours
+    const formattedMinutes = (minutes < 10 ? '0' : '') + minutes
+    const formattedSeconds = (seconds < 10 ? '0' : '') + seconds
+    const formattedTime = `${formattedHours}:${formattedMinutes}:${formattedSeconds}`
+    document.getElementById('clock').innerHTML = formattedTime;
```

Comment:
This if my first time seeing the `.innerHTML` function.  I learned a thing!  Thanks!!